clang/include/clang/AST/DeclCXX.h

```diff
@@ -1139,6 +1139,9 @@ class CXXRecordDecl : public RecordDecl {
   /// \note This does NOT include a check for union-ness.
   bool isEmpty() const { return data().Empty; }
 
+  void setInitMethod(bool Val) { data().HasInitMethod = Val; }
```

Comment:
Could you please add a comment here?
Example: https://github.com/intel/llvm/blob/4d42a5c2c5eb3d77f720cb298691e165e768079e/clang/include/clang/AST/DeclCXX.h#L111